src/video_core/renderer_base.h

```diff
@@ -42,6 +42,8 @@ class RendererBase : NonCopyable {
 
     [[nodiscard]] virtual RasterizerInterface* ReadRasterizer() = 0;
 
+    [[nodiscard]] virtual std::string_view GetDeviceVendor() = 0;
```

Comment:
```suggestion
    [[nodiscard]] virtual std::string_view GetDeviceVendor() const = 0;
```

Perhaps this should return a std::string as opposed to a std::string, given it's in a virtual interface, to ensure no renderer potentially returns a dangling string, regardless of any refactoring the takes place in the future